src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1522,6 +1522,8 @@ protected function handle_terms( $post_id, $request ) {
 				return $result;
 			}
 		}
+
+		return null;
```

Comment:
I could go either way. I opted for adding a `null` return because the return value of this method is used in a few places, for example https://github.com/johnbillion/wordpress-develop/blob/f9a8d730ce25f1478bcba8855b1af4025bfb0c50/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L695, which essentially means a void return ends up being null anyway.